src/Microsoft.Health.Fhir.Core/Exceptions/DeadlockVictimException.cs

```diff
@@ -0,0 +1,24 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Diagnostics;
+using Microsoft.Health.Fhir.Core.Models;
+
+namespace Microsoft.Health.Fhir.Core.Exceptions
+{
+    public class DeadlockVictimException : FhirException
```

Comment:
Perhaps `TransactionDeadlockException` or `StorageDeadlockException`, thoughts?

Reply:
revised to TransactionDeadlockException